liwords-ui/src/gameroom/scss/gameroom.scss

```diff
@@ -686,43 +701,62 @@ p.streak-loss {
     }
     */
   }
+  @include userTile() {
+    border: 0px solid transparent; // Fallback if border is a gradient
+    border: 1px solid ut($tile-last-background);
+    background: ut($tile-last-background);
+  }
   &.blank {
     p.rune {
       @include colorModed() {
         background-color: m($tile-last-blank);
         color: m($tile-last-background);
-
+      }
+      @include userTile() {
+        color: ut($tile-blank-text); // Fallback if border is a gradient
+        color: ut($tile-last-background); // Fallback if border is a gradient
```

Comment:
does this concatenate both `color` settings? Not sure how this works in SCSS.

Reply:
In CSS only the last  valid style counts. In this case, if the tile-last-background is a gradient or picture or something else that's not valid for text (or border), it will use the one before. Metallic is special, but when we do Kickstarter colorways we may even have backgrounds that are images. In which case fallback. I'll remove that second comment. The first one is the fallback.